Foundation-delegation-program/validator-profiles/High Stakes/highstakes.md

```diff
@@ -1,18 +1,29 @@
-We'll comply with each point.
+# Aura Network Validator Pledge
 
-1. Top 25 validators must run an Euphoria testnet node beside Xstaxy mainnet node
-2. The hardware for mainnet meets the recommended baseline
-3. Uptime of at least 90%
-4. Commission rate is greater than 3% and smaller or equal to 5%
-5. Commit to participate in network governance at least 2/3 of the total proposals throughout the delegation period
-6. Must set Gas Price >= 0.001 uaura
-7. Publicly announced to the social channels on the delegation decision
+As the Aura foundation delegation **Round 3** receiver, **High Stakes** pledges to meet the following works. If, for any reason, we do not meet these commitments then we understand that we may lose delegation staked to us.
+
+    1. Top 15 validators must run an Euphoria testnet node beside Xstaxy mainnet node
+    2. The hardware for mainnet meets the recommended baseline    
+    3. Uptime of at least 90%
+    4. Commission rate is greater than 0.1% and smaller or equal to 10%
+    5. Commit to participate in network governance at least 2/3 of the total proposals throughout the delegation period
+    6. Must set Gas Price >= 0.001 uaura
+    7. Publicly announced to the social channels on the delegation decision
+    8. Commit to network contributions listed below: 
+    Category 1: Infrastructure
+    - Provide peers, snapshot, relayers, ...
+    Category 2: Software engineering
+    - Contribute to Aura source code, ...
+    Category 3: Community growth
+    - Creating contents, events, maintaining comunity ...
+    Category 4: Business development
+    - Growing business in NFT, partnerships, ...
 
 Category 1: Infrastructure
 
         --> a map of nodes: https://tools.highstakes.ch/geoloc/xstaxy
 
-        --> we're still up to run relayers -- please feel free to get in touch with any requirements!
+        --> public API & RPC : https://umee-rpc.highstakes.ch:26657, https://umee-api.highstakes.ch:1317
```

Comment:
is it umee endpoints?

Reply:
ha
sorry, let me correct that, got mixed up...